crates/sui-indexer/src/apis/read_api.rs

```diff
@@ -169,15 +169,21 @@ where
             .await
     }
 
-    async fn get_object(&self, object_id: ObjectID) -> RpcResult<GetObjectDataResponse> {
-        self.fullnode.get_object(object_id).await
+    async fn get_object_with_options(
```

Comment:
nit: can just call this get_object?

Reply:
Yes this can be easily done. But there are too many functions that are called `get_object`  in the repo, I think having a slightly different name is really helpful to distinguish the RPC implementation from other implementations. The external interface(i.e., the RPC method name) is still `sui_getObject`.